src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration.mdx

```diff
@@ -856,6 +1109,17 @@ The `package.json` defines `dependencies` as both a local module (for example, `
     ```
   </Collapser>
 
+  <Collapser
+    id="podman"
+    title="Podman"
+  >
+    For podman, launch SJM mounting the directory at `/var/lib/newrelic/synthetics/modules`. In the of SELinux Context volume is mounted with ':z' or ':Z'. For example:
```

Comment:
> In the of SELinux Context volume is mounted with ':z' or ':Z'.
maybe we should mention the reason behind this difference?

Reply:
added the reference document.